src/shared/components/ErrorAlert/ErrorAlert.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useCallback } from 'react';
+import { Alert, AlertTitle, IconButton, Snackbar } from '@mui/material';
+import { Close } from '@mui/icons-material';
+import { useDispatch, useSelector } from 'react-redux';
+import { errorAlertToggle } from './actions/errorAlertActions';
+import type { Dispatch } from 'redux';
+import type {
+  ErrorAlertAllActions,
+  ErrorAlertState,
+} from './types/errorAlertTypes';
+import type { StoreState } from '../../../store';
+import './ErrorAlert.scss';
+
+export const ErrorAlert = () => {
+  const { message, isOpened } = useSelector<StoreState, ErrorAlertState>(
+    state => state.errorAlert,
+  );
+  const dispatch = useDispatch<Dispatch<ErrorAlertAllActions>>();
+
+  const handleToggle = useCallback(() => {
+    if (isOpened) {
+      dispatch(errorAlertToggle());
+    }
+    return;
+  }, [isOpened]);
```

Comment:
1. why do you need return here? 
2. why do you need this logic here?
3. why do you use useCallback?

Reply:
1) fixed 
2) to check after action onClose into alert and auto hide do not toggle it 
3) If message will change , I do not want after re-render to recreate handleFunction which put into onClose and onClick